packages/sui-pde/src/hooks/useDecision.js

```diff
@@ -39,20 +39,24 @@ export default function useDecision(name, {attributes, trackExperimentViewed, qu
         return {enabled: true, flagKey: name, variationKey: forced}
       }
 
+      const notificationId = pde.addDecideListener({
+        onDecide: ({type, decisionInfo: decision}) => {
+          const {ruleKey, variationKey, decisionEventDispatched} = decision
+
+          if (type === 'flag' && decisionEventDispatched) {
```

Comment:
This will make sure that `experimentViewed` is only sent when the feature test has a linked experiment